Tests/SwiftFormatTests/PrettyPrint/AllowWhitespaceOnlyLinesTests.swift

```diff
@@ -0,0 +1,99 @@
+import SwiftFormat
+
+final class AllowWhitespaceOnlyLinesTests: PrettyPrintTestCase {
+  func testAllowWhitespaceOnlyLinesEnabled() {
+    let input =
+      """
```

Comment:
In this file, can you replace any trailing whitespace with literal `\u{0020}` escape sequences? That makes it clear to the reader what's being tested, and we don't risk the spaces being lost if someone's editor strips trailing spaces.

Reply:
Oh, that part was exactly what I was struggling with, and I didn’t know there was such a great solution! Thank you!